### PR TITLE
Add flag if to show borders on case list fields

### PR DESCRIPTION
**Pull request by MartinRiese**
## Product Description

Add flag to case list fields to show a border

## Technical Summary

https://dimagi-dev.atlassian.net/browse/USH-3823

### Safety story

Only additive

### Special deploy instructions

- [x] This PR can be deployed after merge with no further considerations.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations.

### Review

- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change.

### Duplicate PR
Automatically duplicate this PR as defined in [contributing.md](https://github.com/dimagi/commcare-core/blob/master/.github/contributing.md).

### CODE REVIEW

File: src/main/java/org/commcare/suite/model/DetailField.java [hunk 1]
```diff
@@ -49,6 +49,8 @@ public class DetailField implements Externalizable {
     @Nullable
     private EndpointAction endpointAction;
 
+    private boolean showBorder;
```
Comment (shubham1g5) [inline src/main/java/org/commcare/suite/model/DetailField.java:52]: Feels like this should be part of [Style](https://github.com/dimagi/commcare-core/blob/master/src/main/java/org/commcare/suite/model/Style.java) instead

**Review by shubham1g5 — CHANGES_REQUESTED** [review]

**Comment by shubham1g5** [comment]
Can we update the [doc](https://github.com/dimagi/commcare-core/wiki/Suite20) with the attribute as well ? Thanks!

**Comment by MartinRiese** [comment]
Updated the doc

**Comment by MartinRiese** [comment]
duplicate this PR

**Pull request merged**